### ISSUE TITLE
Redundant `.` character when converting `enum`

### ISSUE THREAD

**Issue by igorwojda** [angelolloqui/SwiftKotlin#112]
**Swift source**
```
enum User: String {
    case jane = "Jane"
    case joe = "Joe"
    
    var uuid: String {
        switch self {
        case .jane:
            return "abc"
        case .joe:
            return "cde"
        }
    }
}
```

**Kotlin Result**
```
enum class User (val rawValue: String) {
    jane("Jane"), joe("Joe");

    companion object {
        operator fun invoke(rawValue: String) = User.values().firstOrNull { it.rawValue == rawValue }
    }
    
    val uuid: String
        get() {
            when (this) {
                .jane -> return "abc"
                .joe -> return "cde"
            }
        }
}
```

**Kotlin Expected result (no dot in the `when` element)**
```
enum class User (val rawValue: String) {
    jane("Jane"), joe("Joe");

    companion object {
        operator fun invoke(rawValue: String) = User.values().firstOrNull { it.rawValue == rawValue }
    }
    
    val uuid: String
        get() {
            when (this) {
                jane -> return "abc"
                joe -> return "cde"
            }
        }
}
```

BTW if we want to follow kotlin naming conventions we should make these enum const names uppercase
https://kotlinlang.org/docs/reference/coding-conventions.html#property-names

**Comment by angelolloqui** [comment]
This one will be fixed in next release (WIP). 

Regarding the naming conventions, I intentionally decided not to modify them. The main reason is that since we do not have type information, we could be breaking code afterwards when transpiling the code that uses it. Besides, code convention transformation could be built as a post process by implementing a `TokenTransformPlugin` that modifies it accordingly. 

If you want to contribute, feel free to create such a plugin (plugins can be enabled or disabled easily by final users)

**Comment by angelolloqui** [comment]
Fixed with #115
